view/templates/email/html.tpl

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional //EN">
+<html>
+<head>
+	<title>{{$banner}}</title>
+	<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
+</head>
+<body>
+	<table style="border:1px solid #ccc">
+	<tbody>
+		<tr>
+			<td colspan="2" style="background:#084769; color:#FFFFFF; font-weight:bold; font-family:'lucida grande', tahoma, verdana,arial, sans-serif; padding: 4px 8px; vertical-align: middle; font-size:16px; letter-spacing: -0.03em; text-align: left;">
+				<img style="width:32px;height:32px; float:left;" src='{{$siteurl}}/images/friendica-32.png'>
+				<div style="padding:7px; margin-left: 5px; float:left; font-size:18px;letter-spacing:1px;">{{$product}}</div>
+				<div style="clear: both;"></div>
+			</td>
+		</tr>
+
+		{{$htmlversion nofilter}}
```

Comment:
The HTML version is encapsulated in a table cell structure?

Reply:
yes .. I extraced it because it already was here. But I can play around with the HTML to better embed it ..

Reply:
The htmlversion is now outside of the table structure.

There's now a first table for the common header (`html.tpl` and a second for specific content (`system/html.tpl`)